.github/workflows/ci.yml

```diff
@@ -150,7 +182,6 @@ jobs:
       - name: Run the tests
         run: |
           invoke docs-snippet-tests 'docs' --up-services --verbose
-
```

Comment:
nit: Can you add back in this newline. Having a newline between jobs helps me scan through these .yml files.